microprot/tests/data/test_pdb_search/NC_000913.3_2.fasta

```diff
@@ -0,0 +1,15 @@
+>gi|556503834|ref|NC_000913.3|_2 # 337 # 2799 # 1 # ID=1_2;partial=00;start_type=ATG;rbs_motif=GGAG/GAGG;rbs_spacer=5-10bp;gc_cont=0.531
```

Comment:
The new NCBI format no longer contains GI. It is now simply `NC_000913.3`. Just let you know in case.